accounting/transferwise/tasks.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2017-2018 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""
+TransferWise tasks.
+"""
+
+from django.conf import settings
+from django.core.mail.message import EmailMessage
+from huey import crontab
+from huey.contrib.djhuey import db_periodic_task
+
+from accounting.transferwise import models
+
+
+@db_periodic_task(crontab(day=settings.TRANSFERWISE_BULK_PAYMENT_DAY, hour="0", minute="0"))
+def generate_bulk_payment_csv():
+    """
+    Send an email to the designated TransferWise Bulk Payment sender containing the generated bulk payment CSV.
+    """
+    # Make the new bulk payment for the sender.
+    bulk_payment = models.TransferWiseBulkPayment.objects.create()
+    bulk_payment.create_payments()
+    bulk_payment_csv = bulk_payment.to_bulk_payment_csv(bulk_payment.payments.all())
+
+    # Make the email, attach the CSV and send.
+    email = EmailMessage(
+        subject='TransferWise Bulk Payment CSV',
+        body='TransferWise Bulk Payment CSV is attached.',
```

Comment:
@UmanShahzad It would be cool to include the date range that this file covers in the subject and/or body of the e-mail. We can get that info directly from `bulk_payment` so it should be straightforward to include.

Reply:
We actually plan on making the bulk payment's date range be from ancient history to now! That means we'd always just get "super old date to now" as the date range. The only way to tell appropriately, is to check the bulk payment's individual payments, and get the min and max of date ranges for all the invoices in them. :p

Reply:
@UmanShahzad Ah, OK :) Maybe the date and time at which the CSV was generated would still be interesting? Feel free to consider this nit-level/polish, though -- no need to keep iterating here to address this.

Reply:
@itsjeyd That would be an option, yeah, but the date the CSV is generated is the same as when the email was sent, since it all happens in one go, so the timestamp of the email would give the same information.